avoda-fe/pages/dashboard/profile.vue

```diff
@@ -29,31 +37,22 @@
 
 <script setup>
 import { ref, onMounted } from 'vue';
-import { useRoute } from 'vue-router';
 import {
   Card,
   CardHeader,
   CardTitle,
   CardDescription,
   CardContent,
 } from '@/components/ui/card';
-import axios from 'axios';
-import { useRuntimeConfig } from '#imports';
 
 const config = useRuntimeConfig();
-const route = useRoute();
 const user = ref(null);
-
+const { get } = useApi();
 const fetchUserProfile = async () => {
   try {
-    const { id } = route.params;
-    const response = await axios.get(
-      `${config.public.BASE_URL}/organizations/${id}/members/profile`,
-      {
-        withCredentials: true,
-      }
-    );
-    user.value = response.data;
+    const response = await get(`${config.public.BASE_URL}/users/profile/me`);
```

Comment:
This should be `get('/users/profile/me')`. The `BASE_URL` has been set in the underlying `get` function from `useApi`.